staging/com.smartnews/package.json

```diff
@@ -0,0 +1 @@
+{"name":"com.smartnews","author":"Emily@thingpedia.stanford.edu","main":"index.js","thingpedia-version":428}
```

Comment:
You'll probably want to reformat this to be more readable, add you real email address.
You should also remove "thingpedia-version", which is bogus.